dev/.scalafmt.conf

```diff
@@ -26,10 +26,5 @@ optIn = {
 danglingParentheses.preset = false
 docstrings.style = Asterisk
 maxColumn = 98
-runner.dialect = scala212
-fileOverride {
-  "glob:**/src/**/scala-2.13/**.scala" {
-    runner.dialect = scala213
-  }
-}
+runner.dialect = scala213
```

Comment:
Nice catch. It seems to work. Is it ready, @panbingkun ?

Reply:
Yes, I have checked it on local and it is ready!
Thank you very much! ❤️